lm_eval/tasks/arabic_leaderboard_complete/README.md

```diff
@@ -0,0 +1,254 @@
+      year={2023},
+      eprint={2309.12053},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+@misc{lighteval,
+  author = {Fourrier, Clémentine and Habib, Nathan and Wolf, Thomas and Tunstall, Lewis},
+  title = {LightEval: A lightweight framework for LLM evaluation},
+  year = {2023},
+  version = {0.3.0},
+  url = {https://github.com/huggingface/lighteval}
+}
+```
+
+### Groups and Tasks
+
+* `alghafa`: A multiple-choice evaluation benchmark for zero- and few-shot evaluation of Arabic LLMs prepared from scratch natively for Arabic.
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+    * You can find the list of the tasks as follows:
+      * `arabic_mcq_exams_test_ar`
+      * `arabic_meta_ar_dialects`
+      * `arabic_meta_ar_msa`
+      * `arabic_multiple_choice_facts_truefalse_balanced_task`
+      * `arabic_multiple_choice_grounded_statement_soqal_task`
+      * `arabic_multiple_choice_grounded_statement_xglue_mlqa_task`
+      * `arabic_multiple_choice_rating_sentiment_no_neutral_task`
+      * `arabic_multiple_choice_rating_sentiment_task`
+      * `arabic_multiple_choice_sentiment_task`
+* `arabic_arc_challenge`: AI2 Reasoning Challenge (ARC) is a multiple-choice question task. The dataset contains only natural, grade-school science questions,
+  written for human tests. The challenge set contains only questions answered incorrectly by both a retrieval-based algorithm and a word co-occurence algorithm. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_arc_easy`: This dataset is the same as `arabic_arc_challenge`, except it is not from the challenge set.
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_boolq`: A true/false questions dataset that contains the columns passage, question, and the answer (i.e., true/false). (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_copa`: Choice Of Plausible Alternatives (COPA) is a multiple-choice question dataset, which involves open-domain commonsense causal reasoning. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_exams`: A question answering benchmark for high school examinations in different school subjects that requires knowledge and reasoning in different languages in multiple domains.
+    * Paper: https://aclanthology.org/2020.emnlp-main.438.pdf
+* `arabic_hellaswag`: The tesk is to choose the next set of sentences, based on the given candidates. The tasks involve reading comprehension and information retrieval challenges
```

Comment:
Let's indicate in the names that these tasks are machine-translated--

`arabic_mt_hellaswag` etc?

Reply:
added indicator for machine translated benchmarks and their files - for both light and full versions

Reply:
perfect, lgtm!